changelog/17140.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+**Plugin Multiplexing**: Ensure v5 backends are reinitialized upon reloads
+```
```

Comment:
I'm not sure if we need a changelog entry for a bug fix in an unreleased feature? 🤔 There is already a features CL entry for this. If we do want one, I'd say it's more of a bug fix than improvement.

Reply:
Ya, I think removing it makes the most sense 👍

Reply:
Removed changelog